Marlin/pins.h

```diff
@@ -39,6 +39,8 @@
 
 #if MB(GEN7_CUSTOM)
   #include "pins_GEN7_CUSTOM.h"
+#elif MB(CNControls_12)
+  #include "pins_CNControls_12.h"
```

Comment:
Likewise, the filename after `pins_` should be in all-caps by convention. So rename this to `pins_CNCONTROLS_12.h`.